package.json

```diff
@@ -39,7 +39,7 @@
     "_fmt": "prettier 'src/**/*.ts' 'tests/**/*.ts' 'examples/**/*.js' 'examples/**/*.ts' 'eslint.config.cjs'",
     "fmt": "pnpm _fmt --write",
     "lint": "eslint '**/*.{cts,mts,ts}'",
-    "test": "pnpm jest ./tests/**/*.test.ts --passWithNoTests",
+    "test": "pnpm jest",
```

Comment:
revert it, because now it goes into all the folders and files, including utils/src/examples folders

Reply:
that is handled by testPathIgnorePatterns